src/python/dxpy/utils/exec_utils.py

```diff
@@ -385,7 +385,7 @@ def _install_dep_bundle(self, bundle):
         if bundle["id"].get("$dnanexus_link", "").startswith("file-"):
             self.log("Downloading bundled file {name}".format(**bundle))
             dxpy.download_dxfile(bundle["id"], bundle["name"])
-            self.run("dx-unpack '{}'".format(bundle["name"]))
+            self.run("dx-unpack '{}'".format(re.escape(bundle["name"])))
```

Comment:
re.escape is not the right thing to use for shell escaping-- see pipes.quote instead (you can then remove the quotes here).